.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,31 @@
+name: Release drafter
+
+# Push events to every tag not containing "/"
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
This will push events on any tag

Reply:
for now its fine - lets separately discuss the tag/release strategy ...

Reply:
Yes, is that not what is intended? In what other scenarios will a tag be pushed?

Reply:
I can remove it. Added that during the conversation on another repo https://github.com/opensearch-project/opensearch-py-ml/pull/27#discussion_r1025780783
Git Documentation: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags